builder/elementwise_test.cc

```diff
@@ -61,16 +61,19 @@ class elementwise_pipeline_builder : public node_visitor {
   }
 
   void visit(const constant* c) override {
-    auto value = std::make_unique<buffer<T, Rank>>();
+    auto* dims = SLINKY_ALLOCA(dim, Rank);
```

Comment:
Rank is a template parameter, so you should just do `slinky::dim dims[Rank]` here.